.github/dependabot.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+updates:
+  - package-ecosystem: "gradle"
+    directory: "/"
+    schedule:
+      interval: "weekly"
+    groups:
+      alldependencies:
+        patterns:
+          - "*"
+        update-types:
+          - "minor"
+          - "patch"
+    reviewers:
+      - "@MikeNeilson"
```

Comment:
@MikeNeilson - is there a CDA GitHub Team? for the opendcs one, I had set the reviewers to the team there

Reply:
Not as of yet. I think.

Reply:
I'll check on it today or early tomorrow though. We can definitely make them just not sure if I need to get Will B involved.